manifests/a/AnyLogic/AnyLogic/Uni/8.7.12/AnyLogic.AnyLogic.Uni.locale.en-US.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.Uni
+PackageVersion: 8.7.12
+PackageLocale: en-US
+Publisher: AnyLogic North America
+PublisherUrl: https://the.anylogic.company/
+# PublisherSupportUrl: 
+PrivacyUrl: https://anylogic.com/upload/privacy_policy/anylogic-privacy-policy.pdf
+# Author: 
+PackageName: Anylogic University
+PackageUrl: https://www.anylogic.com/
+License: Proprietary
+LicenseUrl: https://anylogic.com/upload/license_agreements/anylogic-model-end-user-agreement.pdf
+# Copyright: 
+# CopyrightUrl: 
+ShortDescription: AnyLogic is a multimethod, java-based simulation modeling tool developed by The AnyLogic Company.
+Description:  AnyLogic is a multimethod, java-based simulation modeling tool developed by The AnyLogic Company. It supports agent-based, discrete event, and system dynamics simulation methodologies.
+Moniker: anylogic
```

Comment:
```suggestion
Moniker: anylogic-university
```

Reply:
Isn't it more convenient to choose the version when user types `winget install anylogic` (as this is more easy to remember) ? With this change, I assume this university version won't show up when user types the above command. Also, with this change, will this version show up when user types `winget search anylogic` ? Should we add anylogic in its tags?